charts/prometheus-rabbitmq-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Rabbitmq metrics exporter for prometheus
 name: prometheus-rabbitmq-exporter
-version: 1.6.0
+version: 1.6.1
```

new feature should raise minor version

```suggestion
version: 1.7.0
```

---

gotcha, thanks! :)